src/utils/notifications.ios.js

```diff
@@ -33,16 +32,18 @@ export const initializeNotifications = (auth: Auth, saveTokenPush: (arg: string)
 
 export const refreshNotificationToken = () => {};
 
-export const handlePendingNotifications = async switchNarrow => {
-  const notification = await PushNotificationIOS.getInitialNotification();
+export const handlePendingNotifications = async (notification, doNarrow) => {
+  console.log('Opened app by notification', notification); //eslint-disable-line
```

Comment:
Do we want to keep the console.log and why?

Reply:
Removed